client/webserver/webserver_test.go

```diff
@@ -1019,3 +1020,61 @@ func testTrade(t *testing.T, async bool) {
 	tCore.tradeErr = tErr
 	ensure(`{"ok":false,"msg":"expected dummy error"}`)
 }
+
+func Test_prepareAddr(t *testing.T) {
+	tests := []struct {
+		name       string
+		addr       net.Addr
+		allowInCSP bool
+		want       string
+	}{{
+		name: "OK: IPv4",
+		addr: &net.TCPAddr{
+			IP:   net.IPv4(127, 0, 0, 1),
+			Port: 7232,
+		},
+		want:       "127.0.0.1:7232",
+		allowInCSP: true,
+	}, {
+		name: "OK: IPv6",
+		addr: &net.TCPAddr{
+			IP:   net.IPv6loopback,
+			Port: 7232,
+		},
+		want:       "127.0.0.1:7232",
+		allowInCSP: true,
```

Comment:
IPv6 loopback shouldn't be translated into something else.  Don't modify the addr and don't allow in csp.

Reply:
> IPv6 loopback shouldn't be translated into something else. 

I was thinking it's better we log `127.0.0.1` instead of `[::1]`, no? The average non-techy user would find `127.0.0.1 ` more appealing in the logs.

Reply:
They are not interchangeable.  Binding to `[::1]` does not mean you are also bound to 127.0.0.1